### DISCUSSION TITLE
Bot saying maps automatically?

### DISCUSSION THREAD

**Discussion by DogBert02** [SDCore/Apex-Stats-Bot#40] [Q&A] [answered]
Is there anyway to have the bot say /map during a map rotation time limit? Like having it announce Worlds Edge when the map rotates, maybe for both arenas and pubs?

IK that there’s it saying “Playing x/y map” but maybe something more public.

**Comment by SDCore** [comment, marked as answer]
It's currently not possible, but it is planned for the future.
